### PR TITLE
Removed clutter files from first `$ jb create <book name>` and fix starting page mushed button.

### PR DESCRIPTION
**Pull request by MantiMantilla**
A few unused files remained in the first 'optimizacion' directory. These were mostly instructions left behind by a test when first creating the book.

The starting page had an issue wherein the button to the first page was mushed to the right. The fix to this meant changing the 'Intro.ipynb' to 'intro.md'. The look is not entirely the same but is close to the original.

### CODE REVIEW

**Comment by MantiMantilla** [comment]
I've also made changes to the README.md. It is now a proper welcome message.

**Comment by MantiMantilla** [comment]
Also added a CI workflow so that the GitHub page gets automatically deployed on project push or pull to master branch.

**Comment by MantiMantilla** [comment]
'_build' folder will no longer be a part of the repository. CI workflow builds and deploys the folder to gh-pages branch.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 RpF es un proyecto piloto de desarrollo de páginas estáticas para el contenido de cursos del departamento de ingeniería industrial de la Universidad de los Andes, liderado por la instructora Alfaima Solano para el curso Principios de Optimización (IIND-2103).
 
-La idea es tener una página accesible por todos con escenarios de optimización que se prestan para la construcción de modelos de programación matemática lineales. Cada enunciado cuenta con una descripción del escenario y, para luego de que el estudiante haya intentado su propia implementación del modelo en _Python_, un modelo posible junto con su respectiva implementación haciendo uso de la librería _Pulp_.
+La idea es tener una página accesible por todos con escenarios de optimización que se prestan para la construcción de modelos de programación matemática lineal-entera. Cada enunciado cuenta con una descripción del escenario y, para luego de que el estudiante haya intentado su propia implementación del modelo en _Python_, un modelo posible junto con su respectiva implementación haciendo uso de la librería _Pulp_.
```
Comment (AlfaimaSB) [inline README.md]: No todos los ejercicios son lineales enteros.

**Review by AlfaimaSB — COMMENTED** [review]
¿Qué beneficio trae el quitar estas dos líneas de código?

**Review by AlfaimaSB — DISMISSED** [review]
Gracias por actualizar este documento.

File: README.md (hunk 1 above)
Comment (MantiMantilla) [reply README.md]: Ya lo cambio para que diga 'lineal o lineal-entera'!

**Comment by MantiMantilla** [comment]
> ¿Qué beneficio trae el quitar estas dos líneas de código?

No se cuales dos lineas diceeeees.

**Pull request merged**